pootle/apps/pootle_app/management/commands/initdb.py

```diff
@@ -33,7 +33,7 @@ class Command(NoArgsCommand):
 
     def handle_noargs(self, **options):
         self.stdout.write('Populating the database.')
-        initdb(options["create_projects"])
+        InitDB().init(options["create_projects"])
```

Comment:
Maybe the naming is misleading and the intention is different, so correct me if I'm wrong, but I find this construct totally counter-intuitive — there's `__init__` for a purpose.